README.md

```diff
@@ -296,3 +296,147 @@ JSArray args = new JSArray();
 // ...
 connection.event("interfaceName", "methodName", args);
 ```
+
+### JSTP compiler
+
+JSTP compiler is a nice feature for better handling of JSTP packets. You can declare interfaces comfortable for concrete handling purposes and use them instead of `ManualHandler`.
```

Comment:
I think it's better to put it smth like
JSTP compiler is a nice feature to ease handling of JSTP packets. You can declare interfaces that correspond to specific API or simple call to avoid writing all boilerplate code to get the arguments out of the packet. JSTP compiler will parse those at compile time and generate implementations.